### PR TITLE
cli: added get_metadata method

### PR DESCRIPTION
**Pull request by atrisovic**
Closes: #40
Signed-off-by: Ana Trisovic ana.trisovic@cern.ch

### CODE REVIEW

File: cap_client/cli/cli.py [hunk 1]
```diff
@@ -63,6 +63,25 @@ def get(ctx, pid):
 
 
 @click.command()
+@click.argument('field')
+@click.option(
```
Comment (annatrz) [inline cap_client/cli/cli.py]: this has to be required field

File: cap_client/cli/cli.py [hunk 2]
```diff
@@ -63,6 +63,28 @@ def get(ctx, pid):
...
+    '-p',
+    help='Get metadata of the deposit with given pid',
+    default=None,
+    required=True,
+)
+@click.pass_context
+def get_metadata(ctx, field, pid):
+    """Retrieve one or more fields in analysis metadata."""
+    try:
+        response = ctx.obj.cap_api.get_metadata_field(field, pid=pid)
```
Comment (annatrz) [inline cap_client/cli/cli.py]: this call is not gonna work

Comment (atrisovic) [reply cap_client/cli/cli.py]: 👍🏻  fixed now

**Comment by atrisovic** [comment]
Fixed now. Eg: `cap-client get_metadata 'user_analysis.basic_script.source.preserved' --pid 887edf16b562444fb96ed05c40397626`

**Pull request merged**